src/app/components/tabBar/index.tsx

```diff
@@ -28,42 +29,54 @@ const Button = styled.button({
 type Tab = 'dashboard' | 'nft' | 'stacking' | 'settings';
 
 interface Props {
-  tab: Tab
+  tab: Tab;
 }
-function BottomTabBar({ tab }:Props) {
+function BottomTabBar({ tab }: Props) {
   const navigate = useNavigate();
 
   const handleDashboardButtonClick = () => {
-    if (tab !== 'dashboard') { navigate('/'); }
+    if (tab !== 'dashboard') {
+      navigate('/');
+    }
   };
 
   const handleNftButtonClick = () => {
-    if (tab !== 'nft') { navigate('/nft-dashboard'); }
+    if (tab !== 'nft') {
+      navigate('/nft-dashboard');
+    }
   };
 
   const handleStackingButtonClick = () => {
-    if (tab !== 'stacking') { navigate('/stacking'); }
+    if (tab !== 'stacking') {
+      navigate('/stacking');
+    }
   };
 
   const handleSettingButtonClick = () => {
-    if (tab !== 'settings') { navigate('/settings'); }
+    if (tab !== 'settings') {
+      navigate('/settings');
+    }
   };
 
+  const showBottomBar = !isInOptions();
+
```

Comment:
I think it's a bit cleaner to rather do:

```
const showBottomBar = !isInOptions();

if (!showBottomBar) return null;

return (
  <RowContainer...
```

Less nesting and no ternary 😋

Reply:
I can do that in my next fix